docs/source/user-guide/troubleshooting.rst

```diff
@@ -298,7 +312,13 @@ Make sure your conda is up-to-date: ``conda --version``
 
 If not, run: ``conda update conda``
 
-Temporarily set your ``ssl_verify`` variable to false, upgrade the requests package, and then
+Try using the operating system certificate store. Set you ``ssl_verify`` variable to ``truststore``
+using the following command::
+
+    conda config --set ssl_verify truststore
+
+If using the OS certificate store does not solve your issue, temporarily set your ``ssl_verify`` variable
+to false, upgrade the requests package, and then
```

Comment:
```suggestion
If using the operating system certificate store does not solve your issue, temporarily
set your ``ssl_verify`` variable to false, upgrade the requests package, and then
```

Reply:
This suggestion is odd without a minimum supported version of the requests package?

Reply:
I agree with you it is odd. However it appeared like that on the troubleshooting guide, so I did not want to change it without reason. If I had to guess the reason behind this "update requests" is that the upgrade of the requests package may pull  an upgraded certificate bundle, that may address the connection issue.